### PR TITLE
add support for playbackURL in UploadResult on video upload

### PR DESCRIPTION
**Pull request by Suraj3240**
#### What does this PR address?

- [x] GitHub issue (Add reference - #87)
- [ ] Refactoring
- [ ] New feature
- [ ] Bug fix
- [ ] Adds more tests

#### Are tests included?

- [ ] Yes
- [x] No

#### Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I ran the full test suite before pushing the changes and all the tests pass.

### CODE REVIEW

File: api/uploader/upload_asset.go [hunk 1]
```diff
@@ -122,9 +122,11 @@ func (u *API) Upload(ctx context.Context, file interface{}, uploadParams UploadP
 	if err != nil {
 		return nil, err
 	}
+	//Parse playback_url from response
```
Comment (const-cloudinary) [inline api/uploader/upload_asset.go:125]: This code is redundant, response parser automatically parses all relevant fields.

File: api/uploader/upload_asset.go [hunk 2]
```diff
@@ -197,6 +199,7 @@ type UploadResult struct {
 	Placeholder           bool                          `json:"placeholder,omitempty"`
 	URL                   string                        `json:"url"`
 	SecureURL             string                        `json:"secure_url"`
+	PlaybackURL		   	  string 						`json:"playback_url,omitempty"`
```
Comment (const-cloudinary) [inline api/uploader/upload_asset.go:202]: Please use spaces instead of tabs

**Review by const-cloudinary — CHANGES_REQUESTED** [review]
@Suraj3240 , thank you for contribution!

Please fix a few minor issues.

File: api/uploader/upload_asset.go (hunk 1 above)
Comment (Suraj3240) [reply api/uploader/upload_asset.go:125]: > This code is redundant, response parser automatically parses all the relevant fields.

So do I need to remove this code (line 126,127)?

Comment (Suraj3240) [reply api/uploader/upload_asset.go:125]: @const-cloudinary

Comment (const-cloudinary) [reply api/uploader/upload_asset.go:125]: yes, you can remove it.
You can also update [this test ](https://github.com/cloudinary/cloudinary-go/blob/74b7a5291c66ecc64545b170b5fbb7c197fc7a15/api/uploader/upload_asset_test.go#L93) and make sure that PlaybackURL is not empty

Comment (const-cloudinary) [reply api/uploader/upload_asset.go:125]: @Suraj3240 ^

**Comment by Suraj3240** [comment]
I have committed the changes but somehow they are taking time to reflect here. ;(

**Review by const-cloudinary — APPROVED** [review]
@Suraj3240, thank you for contribution!

**Comment by Suraj3240** [comment]
Also can you label it as `hacktoberfest-accepted` so that this will be counted in my PRs.

Also can you update to me regarding the swags-kit which was guaranteed after successful PR merging? 😆

**Comment by colbyfayock** [comment]
@Suraj3240 the repository is labeled with `hacktoberfest` which should be good enough for being counted, are you not seeing it appear? it may take a bit. if it doesnt, you can report back

congrats on the merged PR! since it's officially October, you've always qualified for Hacktoberfest swag! please email [hacktoberfest@cloudinary.com](mailto:hacktoberfest@cloudinary.com) where I'll follow up to request more information for getting you your swag

https://cloudinary.com/blog/hacktoberfest-celebrate-open-source-sdks

**Comment by Suraj3240** [comment]
> @Suraj3240 the repository is labeled with `hacktoberfest` which should be good enough for being counted, are you not seeing it appear? it may take a bit. if it doesnt, you can report back
> 
> congrats on the merged PR! since it's officially October, you've always qualified for Hacktoberfest swag! please email [hacktoberfest@cloudinary.com](mailto:hacktoberfest@cloudinary.com) with your github username and a link to your contribution where I'll follow up to request more information for getting you your swag
> 
> https://cloudinary.com/blog/hacktoberfest-celebrate-open-source-sdks

Hey, thanks for the clarification :)
Much happy to help the community.
`hacktoberfest` label is visible to me and it has been reflected in my account of Hacktoberfest, so no worries. Thanks.

**Pull request merged**